flight/extract_gps.py

```diff
@@ -0,0 +1,43 @@
+from collections import namedtuple
+import json
+
+def extract_gps():
```

Comment:
Need to add type hinting for function return.

See: https://missourimrr.github.io/docs/python/type-annotations/full-guide.html

Reply:
I think I added this correctly in https://github.com/MissouriMRR/SUAS-2023/pull/21/commits/58b0905f0a12ee3e04508c8406fde13da868a233, let me know if something is wrong with it since I'm not too familiar with type hinting